### ISSUE TITLE
RVM 1.28.0 is available

### ISSUE THREAD

**Issue by artem-sidorenko** [rvm/ubuntu_rvm#4]
```bash
Warning, new version of rvm available '1.28.0', you are using older version '1.27.0'.
```

Could you please update the PPA?

**Comment by raelgc** [comment]
Hi @artem-sidorenko and thanks for raising this issue. I'm updating the package now :)

**Comment by raelgc** [comment]
Fixed in commit 7a1bb4e. Package is building in Launchpad now and it'll be available in few minutes.
